Adapter/SymfonyCache.php

```diff
@@ -65,8 +70,9 @@ class SymfonyCache implements CacheAdapterInterface
      * @param bool            $phpCodeCacheEnabled If true, will clear APC or PHP OPcache code cache
      * @param array           $types               A cache types array
      * @param array           $servers             An array of servers
+     * @param array           $timeouts            An array of timeout options
      */
-    public function __construct(RouterInterface $router, Filesystem $filesystem, $cacheDir, $token, $phpCodeCacheEnabled, array $types, array $servers)
+    public function __construct(RouterInterface $router, Filesystem $filesystem, $cacheDir, $token, $phpCodeCacheEnabled, array $types, array $servers, array $timeouts)
```

Comment:
The service would not be BC break but the class will be.

To have a perfect BC, we should make it optional with a warning like said on https://github.com/sonata-project/SonataCacheBundle/pull/115/files#r76977312.